dev/releasing.md

```diff
@@ -0,0 +1,28 @@
+# Tailing Sidecar Release Instruction
+
+1. Prepare Release pull request with Helm Chart version change in [Chart.yaml](helm/tailing-sidecar-operator/Chart.yaml)
+
+2. Create the release tag for commit with Helm Chart version change, e.g.
+
+   ```bash
+   git tag -a v0.1.0 -m "Release v0.1.0" 
+   ```
+
+3. Push the release tag, e.g.
+
+   ```bash
+   git push origin v0.1.0
+   ```
+
+4. For major and minor version change prepare release branch, e.g.
+
+    ```bash
+    git checkout -b release-v0.1
+    git push origin release-v0.1
+    ```
```

Comment:
```suggestion
   ```bash
   git checkout -b release-v0.1
   git push origin release-v0.1
   ```
```

Reply:
Instead of aligning it all to 3 spaces I'd maybe move all code blocks to indentation of 4 spaces

Reply:
imho 3 spaces is more readable with ordered list

```
1. element 1
   text related to el 1
```

vs 
```
1. element 1
    text related to el 1
```

Reply:
😬